src/AI-EditDistances/AILevenshteinDistance.class.st

```diff
@@ -30,19 +30,17 @@ AILevenshteinDistance >> distanceBetween: firstString and: secondString [
 { #category : 'private' }
 AILevenshteinDistance >> distanceMatrixBasedOn: firstString and: secondString [
 
-	| distanceMatrix |
-	"Changed from CTArray2D to Array2D to fix undeclared variable error"
-	distanceMatrix := Array2D
-		rows: secondString size + 1
-		columns: firstString size + 1.
+    | distanceMatrix |
+    distanceMatrix := CTArray2D
+        extent: (firstString size + 1) @ (secondString size + 1).
```

Comment:
I would say to instantiate the array 2d with the method `#rows:columns:` instead, for clarity